agr_java_core/src/main/java/org/alliancegenome/neo4j/repository/AlleleRepository.java

```diff
@@ -440,7 +440,7 @@ private String getCypherQuery(String relationship) {
     }
     public Map<String, Allele> getAllAlleleVariants() {
 
-        String query = "MATCH p1=(:Species)-[:FROM_SPECIES]-(a:Allele)";
+        String query = "MATCH p1=(:Species)-[:FROM_SPECIES]-(a:Allele)--(g:Gene)";
```

Comment:
I don't know if this is a long running query, but we have found its better to specify the relationship between the nodes to speed up the query so in this case I think it should be a `(a:Allele)-[:IS_ALLELE_OF]-(g:Gene)`

Reply:
@oblodgett fixed neo query as suggested.